qutebrowser/config/config.py

```diff
@@ -51,26 +51,26 @@
 
 class change_filter:  # noqa: N801,N806 pylint: disable=invalid-name
```

Comment:
I was going to ask you about this. It was needed back then but at one point someone removed this dependency. I kept it around because it looked like a potentially useful change, although now it doesn't fit in this PR. I'll remove it.